components/library/gegl/Makefile

```diff
@@ -10,79 +10,73 @@
 #
```

Comment:
gegl needs the updated babl:
meson.build:159:0: ERROR: Invalid version of dependency, need 'babl' ['>=0.1.78'] found '0.1.18'.